### DISCUSSION TITLE
Fraction metabolized for 'last dose' at steady state

### DISCUSSION THREAD

**Discussion by nina-nauwelaerts** [Open-Systems-Pharmacology/Forum#1263] [Questions & Problems]
Hi, I was wondering if there is a way to see the fractions for the last dose only, when simulating multiple doses (steady state).
If I use the "total fraction of dose", I see the cumulative fraction metabolized via each enzyme (from all doses). In literature, I have the fractions metabolized via each enzyme at a certain time point for a 'single' dose given at steady-state. However, in this study, there is also a fraction of the parent medicine (i.e. not metabolized); and there is auto-induction for some enzymes, thus I cannot compare to the 'total' fraction metabolized over all doses.
Or is that only possible if I create a new observer for this in MoBi?

**Comment by Yuri05** [comment]
>Or is that only possible if I create a new observer for this in MoBi? 

Yes. Here one possible solution: [FractionLastDose.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/9183298/FractionLastDose.zip)

1. Add parameters "**Total amount**" and "**Total amount before last application**" (const zero) to all metabolites of interest

![grafik](https://user-images.githubusercontent.com/25061876/180829372-93a09c6a-e2ab-4e9b-a3ec-1e4a961f87c0.png)

2. In the last application: set `Total amount before last application = Total amount` for each metabolite (make sure to activate "`Use as value`" option.)

![grafik](https://user-images.githubusercontent.com/25061876/180831030-368e6ecb-51cc-4fe7-91f2-86777b86a107.png)

3. Create the new observer "**Total fraction of last dose**" and define it as 

`(Total metabolite amount - Total metabolite amount before the last application ) / Drug mass of the last application`

![grafik](https://user-images.githubusercontent.com/25061876/180831675-ddf3938a-8c5b-4d2f-93ee-a844eaee3448.png)

5. Voilà

![grafik](https://user-images.githubusercontent.com/25061876/180834374-b759ab03-be38-4c89-9124-e7ba8dd215c5.png)
